README.md

```diff
@@ -41,11 +42,11 @@ Steps:
     - OIDC Claims:
         - Email is included automatically without being set here.
         - `preferred_username` will map to the users name + discrim e.g. `Erisa#9999`
-        - If the Auth URL is `/guilds` then the `guilds` claim can be used toprovide a list of guild IDs.
+        - If the Auth URL is `/guilds` then the `guilds` claim can be used to provide a list of guild IDs.
         - Anything else from here will work: https://discord.com/developers/docs/resources/user#user-object-user-structure
 - See the Examples section below for help with constructing policies.
 
-## Usage with roles
+## Usage with roles (Bot required)
```

Comment:
TODO: Note about ratelimits for everything and add a link to the cached roles section.